### PR TITLE
Create comment repo

### PR DESCRIPTION
**Pull request by TDM1911**
## Related Tickets:
- [#31851](https://edu-redmine.sun-asterisk.vn/issues/31851)

## WHAT this PR does?
- Implement Comment Repository

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Review by kienpham9819 — APPROVED** [review]

**Comment by TDM1911** [comment]
ready

File: app/Repositories/Comment/CommentRepositoryInterface.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Repositories\Comment;
+
+interface CommentRepositoryInterface
```
Comment (huyenttm-0901) [inline app/Repositories/Comment/CommentRepositoryInterface.php]: extends base interface nhé

**Comment by TDM1911** [comment]
ready

**Comment by TDM1911** [comment]
ready

**Pull request merged**